core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -664,9 +621,5 @@ private static String toNormalizedIconUrl(String url) {
         icons.addIcon(new Icon("icon-terminal icon-xlg", "48x48/terminal.png", Icon.ICON_XLARGE_STYLE));
         icons.addIcon(new Icon("icon-user icon-xlg", "48x48/user.png", Icon.ICON_XLARGE_STYLE));
         icons.addIcon(new Icon("icon-warning icon-xlg", "48x48/warning.png", Icon.ICON_XLARGE_STYLE));
-        icons.addIcon(new Icon("icon-yellow icon-xlg", "48x48/yellow.png", Icon.ICON_XLARGE_STYLE));
-
-        // Capture a list of the core icons.
-        icons.coreIcons.putAll(icons.iconsByCSSSelector);
```

Comment:
@fqueiruga is this intentionally removed?

Reply:
looks like the code is unused =/.

Can either re-add for compat, leave empty, deprecate, or remove any thoughts?

Reply:
going to just re-add for now